### DISCUSSION TITLE
Error Updating cache to install MetaFieldsBundle

### DISCUSSION THREAD

**Discussion by fsimonENZ** [kimai/kimai#4103] [Plugins]
Hello,

I'm trying to install MetaFieldsBundle Plugin, but I get an error when I try to reload the cache.

I'm following the steps in the link: https://www.kimai.org/store/custom-fields-bundle.html

First of all, 

Copy files
Extract the ZIP file and upload the included directory and all files to your Kimai installation to the new directory:

var/plugins/MetaFieldsBundle/
The file structure needs to look like this afterwards:

var/plugins/
├── MetaFieldsBundle
│   ├── MetaFieldsBundle.php
|   └ ... more files and directories follow here ...

But when I trying to reload the kimai cache with command: bin/console kimai:reload --env=prod I get this error: 

[ERROR] There is no extension able to load the configuration for "fos_user" (in "/var/www/kimai2/config/packages/fos_user.yaml"). Looked for namespace "fos_user", found ""framework", "doctrine", "doctrine_migrations", "sensio_framework_extra", "monolog", "twig", "security", "admin_lte", "jms_serializer", "fos_rest", "nelmio_api_doc", "nelmio_cors", "webpack_encore", "twig_extra", "handcraftedinthealps_rest_routing", "meta_fields", "kimai"".

I also try to rebuilt cache manually but if I do that I loss my installation.

Any ideas?

**Comment by kevinpapst** [comment]
You have a broken installation, which is in no way related to the plugin.
If you run `git status` you will see a lot of warnings I assume.
The error is right in front of you: 

> /var/www/kimai2/config/packages/fos_user.yaml

This file should not be there since ages.

**Reply by fsimonENZ** [reply]
And is there any way to resolve it other than reinstalling it?

**Reply by kevinpapst** [reply]
Maybe 🤷 or most certainly, but that needs a certain knowledge of Kimai.

Simpler: Create a backup first! Then delete the Kimai directory, reinstall the same version at the same place and then just replace the .env file, which holds secret and database credential.

Please understand that I cannot give unlimited free support for such a task.

**Reply by fsimonENZ** [reply]
Ok, thank you

**Comment by fsimonENZ** [comment]
I finally reinstall the Kimai Software and it is working the plugin for Kimai version 1.25.0, but I would like to update to last version 2.0.26. I bought the plugin on November and the last bundle version i have is 1.24.3. Where I could get the 2.0.7 bundle version for Kimai 2.0.26? Thanks in advance

**Reply by kevinpapst** [reply]
You should have received a newsletter regarding that issue. Contact support@kimai.org if not
